tests/unit/components/Questions/Palette/QuestionPalette.spec.ts

```diff
@@ -1,6 +1,55 @@
 import { mount } from "@vue/test-utils";
 import QuestionPalette from "@/components/Questions/Palette/QuestionPalette.vue";
 
+describe("New buttons in question palette", () => {
+  const wrapper = mount(QuestionPalette);
+  it("Buttons are visible or not", () => {
+    expect(wrapper.find('[data-test="toggleInstructions"]').exists()).toBeTruthy();
+    expect(wrapper.find('[data-test="togglePalette"]').exists()).toBeTruthy();
+  })
+
+  it("By default palette should be visible", async () => {
+    expect(wrapper.find('[data-test="instruction-page"]').exists()).toBe(false);
+    expect(wrapper.find('[data-test="question-palette"]').exists()).toBe(true);
+  })
+
+  it("when instruction button is clicked", async () => {
```

Comment:
`it(description, code)`

description should explain what is expected out of the test. 

so the description here should be, "instructions are visible when instruction button is clicked"

change description for other tests too accordingly. description should explain what to expect